gnovm/pkg/gnolang/machine.go

```diff
@@ -881,10 +888,17 @@ const (
 	OpReturnCallDefers  Op = 0xD7 // TODO rename?
 )
 
+const GasFactorCpu int64 = 1
+
 //----------------------------------------
 // "CPU" steps.
 
 func (m *Machine) incrCPU(cycles int64) {
```

Comment:
incrCPU is fine, we generally leave the caps as is. 
But gasFactorCpu should be renamed to gasFactorCPU.
grep becomes tricky without this kind of consistency.

Reply:
sure.